### ISSUE TITLE
Add prioritized experience replay

### ISSUE THREAD

**Issue by r7vme** [r7vme/learning-to-drive-in-a-day#1]
Right now policy learning process is vague and does not improve over time (i.e. good policy can be learned in 3 episodes, but after 10 episodes policy can degrade completely).

So after spending time adjusting [optimal buffer size and optimization steps](https://github.com/r7vme/learning-to-drive-in-a-day/blob/4ec9b04fbee47ef09be0c93752a14912e253a1e2/run.py#L63-L64), i see that it's pretty random. I assume that [prioritized experience buffer](https://github.com/r7vme/learning-to-drive-in-a-day/blob/4ec9b04fbee47ef09be0c93752a14912e253a1e2/run.py#L63-L64) for DDPG will help imrove situation.

In short, it will make sure
- to sample unseen observations (by using infinite priority)
- to sample "valuable" observations (by computing priority based on TD error)
